python/ray/tune/examples/pbt_word_language_model_example.py

```diff
@@ -0,0 +1,534 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Train word-level language modeling RNN on Wikitext-2 using Population Based
+Training. Perturbs learning rate and drop rate.
+
+Dataset:
+https://www.salesforce.com/products/einstein/ai-research/the-wikitext-dependency-language-modeling-dataset/.
+
+Adapted from https://github.com/pytorch/examples/tree/master/word_language_model.
+"""
+import argparse
+from io import open
+import math
+import os
+import time
+
+import numpy as np
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+import ray
+from ray.tune import Trainable, run
+from ray.tune.schedulers import PopulationBasedTraining
+
+############################### DATA ###############################
+class Dictionary(object):
+    def __init__(self):
+        self.word2idx = {}
+        self.idx2word = []
+
+    def add_word(self, word):
+        if word not in self.word2idx:
+            self.idx2word.append(word)
+            self.word2idx[word] = len(self.idx2word) - 1
+        return self.word2idx[word]
+
+    def __len__(self):
+        return len(self.idx2word)
+
+
+class Corpus(object):
+    def __init__(self, path):
+        self.dictionary = Dictionary()
+        self.train = self.tokenize(os.path.join(path, 'train.txt'))
+        self.valid = self.tokenize(os.path.join(path, 'valid.txt'))
+        self.test = self.tokenize(os.path.join(path, 'test.txt'))
```

Comment:
Also, can you make sure you do the linting?

https://ray.readthedocs.io/en/latest/tune-contrib.html